### DISCUSSION TITLE
jwtVerify failed

### DISCUSSION THREAD

**Discussion by DophinL** [logto-io/logto#3872] [Q&A]
Error message:
JWSInvalid: Invalid Compact JWS

![image](https://github.com/logto-io/logto/assets/8707316/c7597c37-131c-4131-b46e-a62352d44f79)

Anyone can help? I tried all ways

**Comment by charIeszhao** [comment]
As the code comment suggests, the audience is the resource indicator of the current API resource, not the web app ID. You should go to "API resources" in console, and create an API resource.
